lib/beaker/hypervisor/openstack.rb

```diff
@@ -120,7 +121,7 @@ def provision
         ip = @compute_client.addresses.find { |ip| ip.instance_id.nil? }
         if ip.nil?
           @logger.debug "Creating IP for #{host.name} (#{host[:vmhostname]})"
-          ip = @compute_client.addresses.create
+          ip = @compute_client.addresses.create({:pool => @options[:pool]})
```

Comment:
No check here for 'nil' @options[:pool] - will want a default value or a reasonable fall through.